src/src/AdminBundle/Resources/views/base_admin.html.twig

```diff
@@ -0,0 +1,37 @@
+{% extends "@AdminBundle/Resources/views/base.html.twig" %}
+
+{% block navigation %}
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
+        <a class="navbar-brand" href="#">Event Manager</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault"
+                aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item">
```

Comment:
toto dame jedneho dna do loopu. alebo teraz? teraz to podla mna nema este vyznam.

Reply:
Podla mna to chceme mat takto a nie v loope, predsalen su tam rozdiele odkazy, rozdiele `if`-y a hlavne je tych veci konstante vela (asi 5?) cize podla mna je to v pohode takto. Loop by len skomplikoval logiku a skratil 30riadkov HTML na 5 HTML a 10 riadkov instanciovania dat do loopu.